content/v2.0/reference/cli/influx/telegrafs/_index.md

```diff
@@ -0,0 +1,46 @@
+---
+title: influx telegrafs
+description: >
+ placeholder
```

Comment:
to be sure, are these still placeholders?

Reply:
Doh, nice catch. I'll update these.